### ISSUE TITLE
Plugin slows startup on fish shell

### ISSUE THREAD

**Issue by ericandrewlewis** [editorconfig/editorconfig-vim#100]
Both vim and neovim run the fish shell configuration script on startup when the editorconfig-vim plugin is installed, which in my case is time-consuming.

Tested on:
```
nvim v0.2.3-dev / Vim 8.0
macOS Sierra 10.12.6
Reproduced on iTerm2 and Apple Terminal
```

### Steps to reproduce using a basic vimrc:

1. Add a slim vim runtime configuration to install editorconfig-vim:

`~/.vimrc`

    ```
    set nocompatible hidden laststatus=2

    if !filereadable('/tmp/plug.vim')
      silent !curl --insecure -fLo /tmp/plug.vim
        \ https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
    endif

    source /tmp/plug.vim
    call plug#begin('/tmp/plugged')
    Plug 'editorconfig/editorconfig-vim'
    call plug#end()

    autocmd VimEnter * PlugClean! | PlugUpdate --sync
    ```

2. Install and switch to the [fish shell](fishshell.com)
3. Add a fish shell runtime configuration which logs the date to a log file

    `date >> ~/fishconfig.log`

4. Tail the log `tail -f ~/fishconfig.log` and keep it open
5. Open up vim to the `~/.vimrc` file

`vim -u ~/.vimrc ~/.vimrc`

and notice that `fishconfig.log` has been touched when vim started up.

If you setup the same logging in a `~/.bashrc`, you'll notice the bashrc isn't touched when opening vim.

Neovim will run the fishrc every time, and also when opening new files. Vim will run the fishrc only when installing the plugin.

**Comment by xuhdev** [comment]
Are you sure this is caused by editorconfig? I don't see any reason to touch an irrelevant file on startup...

**Comment by ericandrewlewis** [comment]
I think so...

if I remove 

```
Plug 'editorconfig/editorconfig-vim'
```

and replace it with another plugin like

```
Plug 'tpope/vim-commentary'
```

I don't see the fishconfig logging to the log file I set up.

After some more testing, I notice this behavior is only with neovim. The fish config file run in vim described above seems to be from the vim-plug setup.

**Comment by ericandrewlewis** [comment]
I traced it to [this line](https://github.com/editorconfig/editorconfig-vim/blob/master/plugin/editorconfig.vim#L494), which is when editorconfig-vim executes the Python script.

**Comment by ericandrewlewis** [comment]
This is probably because of how Neovim implements `system()` https://github.com/neovim/neovim/issues/6015

**Comment by xuhdev** [comment]
What is your `g:EditorConfig_core_mode`?

**Comment by cxw42** [comment]
This should be fixed with the latest plugin (1.1.1), which by default does not use an external core.  Please feel free to comment back and reopen if you are still having this problem with the latest plugin.
